tests/test_jinja_report.py

```diff
@@ -68,7 +68,8 @@ def test_jinja_report_attachment(self):
             extension='html',
             template_extension='html',
             report_content_custom=(
-                b"{{ attachments(record, 'example.png') | b64encode }}"
+                b"{{ attachments(record, 'example.png') "
+                b"| b64encode | decode }}"
```

Comment:
There is really no need to register a filter nor additional functions in the context since jinja2 templates can already call the decode method of the resulting bytes object
```
b"{{ (attachments(record, 'example.png') | b64encode).decode() }}"
```

Maybe update the unittest as a showcase but the additional filter is unnecessary, jinja2 is powerful enough as it is.